kalite/testing/mixins/browser_mixins.py

```diff
@@ -316,25 +324,31 @@ def browser_login_user(self, username, password, facility_name=None, browser=Non
         """
         browser = browser or self.browser
 
-        login_url = self.reverse("login")
+        login_url = self.reverse("homepage")
```

Comment:
Why go through all this rigamarole when we can just use the new API endpoint you made? (This is pretty brittle, unfortunately.) Same goes for logout. Working on something in a PR, along with some screenshot fixes.